### ISSUE TITLE
Please consider moving it under Processing

### ISSUE THREAD

**Issue by pcav** [havatv/qgismultidistancebufferplugin#7]
You'll save coding the interface, and the user will be able to integrate this into more complex workflows, having iterations, and much more.
Ask @volaya for howto, or have a look to other Processing plugins.

**Comment by havatv** [comment]
It would be nice to have Processing plugins described properly in the official QGIS documentation.  I have not been able to find anything useful for plugin developers.  Looking at other Processing plugins is an option, but I was hoping to get an even better understanding of this before starting.

**Comment by pcav** [comment]
Hopefully we'll have instructions soon, thanks to Victor.

**Comment by kannes** [comment]
Processing has a "Multi-ring buffer (constant distance)" tool nowadays that will create the non-overlapping parts including a `ringId` value denoting the number of the ring (1 is the central one) and a `distance` value for the maximum distance of each ring (e.g. for rings up to 10 units, the outermost ring will have a 10).

![image](https://user-images.githubusercontent.com/7661092/152555377-9b8a8a02-8e8f-47a9-8f06-a9579e4ed22f.png)

The plugin still offers more, e.g. the varying distances per ring.
